### ISSUE TITLE
Issue with Species Names

### ISSUE THREAD

**Issue by conneelyb** [zooniverse/wildcam-gorongosa#203]
A user reported that they were using WildCam on a PC on Sunday and the species names began appearing as odd names like "Moose" and "Flame" (see attached screenshot). It seems like an isolated incident and I don't have any specs on the computer, except that it is a PC and they were using Chrome. If need be, I can try to get some more info on the computer.

![img_5299](https://cloud.githubusercontent.com/assets/14139352/13087438/f12780c2-d4b5-11e5-9b7b-bef9deffc84c.JPG)

**Comment by alexbfree** [comment]
That's really weird. Is he able to reproduce this? Is there a particular Subject that this always happens with? If so can we have the subject ID (or talk URL)?

**Comment by conneelyb** [comment]
This may be a dead end - we used a set of rented PC laptops for an event this weekend so we can't reproduce it on the same machine. The names only appeared when the species list was filtered, and the same erroneous names appeared consistently for the same species over and over. This did occur on multiple images. It may have been some sort of issue with that machine's operating system. We haven't seen the issue on any other machine (we just tried it on another PC running chrome), so I wouldn't worry about it unless we see it happen again. Just wanted to let you know..
